### PR TITLE
doc(wit): add README for wasmcloud:postgres

### PR DESCRIPTION
**Pull request by vados-cosmonic**

### CODE REVIEW

File: wit/postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# 🐘 `wasmcloud:postgres` WIT interface
+
+This folder contains [WIT][wit] definitions for `wasmcloud:postgres`, an interface for interacting with the [Postgres database system][postgres] from WebAssembly.
+
+[wit]: https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md
+[postgres]: https://postgresql.org
+
+## 👟 Using this WIT interface
+
+While `wasmcloud:postgres` is best used from the [wasmCloud][wasmcloud] ecosystem (i.e. with the wasmCloud `sqldb-postgres` provider), this WIT contract can be used by any ecosystem project that supports WIT interfaces and fulfilling WIT contracts (whether locally, or in a distriuted fashion as wasmCloud does).
```
Comment (thomastaylor312) [inline wit/postgres/README.md:10]: Spelling: distributed

File: wit/postgres/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+[wit-bindgen-rust]: https://github.com/bytecodealliance/wit-bindgen
+
+### ⬇️ Downloading this WIT
+
+While ecosystem tooling for pulling and using WIT-manifest-only components develops, the easiest way to get started is to use [`wit-deps`][wit-deps], a dependency manager for WIT files.
+
+In your project, include the following `wit/deps.toml`:
+
+```yaml
+postgres = "https://github.com/vados-cosmonic/wit-wasmcloud-postgres/archive/v0.1.0-draft.tar.gz"
```
Comment (thomastaylor312) [inline wit/postgres/README.md:41]: Should this be the new location as a release asset?

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
Two minor issues, but otherwise looking real good

File: wit/postgres/README.md (hunk 1 above)
Comment (vados-cosmonic) [reply wit/postgres/README.md]: Good catch, fixed!

File: wit/postgres/README.md (hunk 2 above)
Comment (vados-cosmonic) [reply wit/postgres/README.md]: Ah, forgot to change this -- done!

**Review by thomastaylor312 — APPROVED** [review]

**Pull request merged**